webdriver/newsession/create-1.py

```diff
@@ -0,0 +1,24 @@
+#META: timeout=long
```

Comment:
Then I agree with @AutomatedTester that it should at least get a better name. Perhaps they can be grouped more intelligently, or in worst case, be split up even further with file names that say something about what the file tests.